src/lib389/lib389/cli_conf/backend.py

```diff
@@ -65,6 +65,10 @@
         'deadlock_policy': 'nsslapd-db-deadlock-policy',
         'db_home_directory': 'nsslapd-db-home-directory',
         'db_lib': 'nsslapd-backend-implement',
+        'lmdb_size': 'nsslapd-mdb-max-size',    # For dscreate (unit is Gb)
```

Comment:
Why have this as two separate parameters? Just because of the different units? Why not do something else to parse the units or similar?

Reply:
I hesitated about that one, but IMHO you are right: parsing human friendly size would be more consistent and that is not hard do. So I will change that.